facts/inventory.py

```diff
@@ -703,6 +703,11 @@ def generatewasgehtconfig(switches, routers, pis, aps, servers, outputdir):
         wasgehtconfig[ap["name"]] = {"address": ap["ipv4"]}
     for server in servers:
         wasgehtconfig[server["name"]] = {"address": server["ipv6"]}
+    wasgehtconfig["google88v6"] = {"address": "2001:4860:4860::8888"}
+    wasgehtconfig["google88v4"] = {"address": "8.8.8.8"}
+    wasgehtconfig["google44v6"] = {"address": "2001:4860:4860::8844"}
+    wasgehtconfig["google44v4"] = {"address": "8.8.4.4"}
+    wasgehtconfig["localhost"] = {"address": "::1"}
```

Comment:
added localhost for the nix checks, also added the upstream Google DNS servers over IPv4 and IPv6 as additional sanity check in the dashboard.